backend/src/src/entities/song.entity.ts

```diff
@@ -5,12 +5,15 @@ export default class SongEntity extends BaseEntity {
   duration: number;
   artist: string;
   genre: string;
+  currently_playing: boolean;
+  times_ever_played: number;
```

Comment:
Pra n quebrar os testes existentes, e ainda considerar q uma musica pode existir sem nunca ser tocada
```suggestion
  times_ever_played?: number;
```